<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="listPorts">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>211</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>Portları Tazele</string>
    </property>
   </widget>
   <widget class="QComboBox" name="ports">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="connectToArduino">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>90</y>
      <width>211</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>Bağlan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="send">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>220</y>
      <width>261</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>Arduinoya Selam Ver</string>
    </property>
   </widget>
   <widget class="QPushButton" name="disconnectFromArduino">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>130</y>
      <width>211</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>Bağlantıyı Kes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="read">
    <property name="geometry">
     <rect>
      <x>27</x>
      <y>220</y>
      <width>261</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>Ardunadan Geleni Oku</string>
    </property>
   </widget>
   <widget class="QLabel" name="incomingData">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>veri yok</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>596</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
